tensorly/decomposition/tests/test_cp.py

```diff
@@ -13,6 +14,29 @@
 from ...testing import assert_array_equal, assert_
 
 
+def tucker_congruence(A, B):
+    As = A/T.norm(A, axis=0)
+    Bs = B/T.norm(B, axis=0)
+
+    return T.dot(T.transpose(As), Bs)
+
+
+def best_congruence(A, B):
```

Comment:
If we do decide to make it more widely accessible, I guess we could also return the optimal permutation?